mysite/w3/library.py

```diff
@@ -165,6 +166,98 @@ def getBonusLevelAdviceGroup() -> AdviceGroup:
     )
     return bonusLevelAdviceGroup
 
+def getCheckoutSpeedAdviceGroup() -> AdviceGroup:
+    checkoutSpeedAdvices = []
+
+    # Meal
+
+    checkoutSpeedAdvices.append(Advice(
+        label=f"Meal: Fortune Cookie: {session_data.account.meals['Fortune Cookie']['Value']}%",
+        picture_class="fortune-cookie",
+        progression=session_data.account.meals["Fortune Cookie"]["Level"],
+        goal=maxMealLevel 
+    ))
+
+    # Atom
+
+    checkoutSpeedAdvices.append(Advice(
+        label=f"""Oxygen - Library Booker: {2*session_data.account.atom_collider['Atoms']["Oxygen - Library Booker"]['Level']}/60%""",
+        picture_class="oxygen",
+        progression=session_data.account.atom_collider['Atoms']["Oxygen - Library Booker"]['Level'],
+        goal=20 + (10 * session_data.account.gaming['SuperBits']['Isotope Discovery']['Unlocked'])
+    ))
+
+    # Tower
+    
+    checkoutSpeedAdvices.append(Advice(
+        label=f"Talent Book Library building: {((session_data.account.construction_buildings['Talent Book Library']['Level']-1) * 5)}/{session_data.account.construction_buildings['Talent Book Library']['MaxLevel']*5}%",
+        picture_class="talent-book-library",
+        progression=session_data.account.construction_buildings['Talent Book Library']['Level'],
+        goal=session_data.account.construction_buildings['Talent Book Library']['MaxLevel']
+    ))
+
+    # Bubble
+
+    checkoutSpeedAdvices.append(Advice(
+        label=f"Ignore Overdues bubble: {session_data.account.alchemy_bubbles['Ignore Overdues']['BaseValue']:.2f}/100%",
+        picture_class="ignore-overdues",
+        progression=session_data.account.alchemy_bubbles['Ignore Overdues']['Level'],
+        resource=session_data.account.alchemy_bubbles['Ignore Overdues']['Material']
+    ))
+
+    # Vial
+
+    vialBonus = session_data.account.alchemy_vials.get('Chonker Chug (Dune Soul)', {}).get('Value', 0) * session_data.account.vialMasteryMulti
+    checkoutSpeedAdvices.append(Advice(
+        label=f"Chonker Chug vial: +{vialBonus:.2f}%",
+        picture_class='chonker-chug',
+        progression=session_data.account.alchemy_vials.get('Chonker Chug (Dune Soul)', {}).get('Level', 0),
+        goal=13
+    ))
+
+    # Stamp
+
+    stampLevel=session_data.account.stamps.get("Biblio Stamp", {}).get('Level', 0)
+    checkoutSpeedAdvices.append(Advice(
+        label=f"Stamp: Biblio Stamp: +{stampLevel}%",
+        picture_class="biblio-stamp",
+        progression=stampLevel,
+        goal=stamp_maxes.get("Biblio Stamp"),
+        resource=session_data.account.stamps.get("Biblio Stamp", {}).get('Material', 0),
+    ))
+
+    # Superbit
+
+    toons = session_data.account.all_characters
+    gaming_levels = []
+    for toon in toons:
+        gaming_levels.append(toon.gaming_level)
+    gaming_level = max(gaming_levels)
```

Comment:
Got you covered: There is a dictionary already parsed for this.
`gaming_level = max(session_data.account.all_skills["Gaming"])`